### PR TITLE
Draft version

### PR DESCRIPTION
**Pull request by dwantula**

### CODE REVIEW

File: .eslintrc.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+module.exports = {
```
Comment (mi-chal) [inline .eslintrc.jsx]: This file can't have `jsx` extension. It can be `.js` or `.json`. Only React components use JSX.

File: debug.log [hunk 2]
```diff
@@ -0,0 +1 @@
+[1116/073045.253:ERROR:directory_reader_win.cc(43)] FindFirstFile: System nie mo�e odnale�� okre�lonej �cie�ki. (0x3)
```
Comment (mi-chal) [inline debug.log]: Remove this file

File: src/App.js [hunk 3]
```diff
@@ -1,25 +1,53 @@
-import logo from './logo.svg';
-import './App.css';
+import React, { PureComponent } from 'react';
```
Comment (mi-chal) [inline src/App.js]: App file should have `jsx` extension

File: src/App.js [hunk 4]
```diff
@@ -1,25 +1,53 @@
...
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+const pageSwitch = [
```
Comment (mi-chal) [inline src/App.js]: Not very fortunate name. I'd change that for pages or screens.

File: src/screens/BeerOfTheDay/.prettierrc [hunk 5]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (mi-chal) [inline src/screens/BeerOfTheDay/.prettierrc]: prettierrc file should live in main folder

File: src/screens/BeerOfTheDay/BeerOfTheDay.jsx [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+import Heading from '../../shared/components/Heading/Heading';
+import RandomBeer from '../../shared/components/RandomBeer/RandomBeer';
+import Button from '../../shared/components/Button/Button';
+import './styles.scss';
+
+class BeerOfTheDayComponent extends PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      value: null,
```
Comment (mi-chal) [inline src/screens/BeerOfTheDay/BeerOfTheDay.jsx]: How about some more meaningful name for this variable?

File: src/screens/BeerOfTheDay/BeerOfTheDay.jsx [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+          type="h3"
+          className="title-beer-day"
+          text="Beer of the day"
+        />
+        <Button 
+          className=" button-choose" 
+          onClick={this.getRandomBeer}
+          text="Choose a beer"
+        />
+        {value ? <RandomBeer value={value}/> : value }
```
Comment (mi-chal) [inline src/screens/BeerOfTheDay/BeerOfTheDay.jsx]: What will be displayed if there is no value?

File: src/screens/BeerOfTheDay/styles.scss [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+  outline: 0;
+  font-family: fantasy;
+  border: none;
+  background-color: white;
+  margin-left: 450px;
+  margin-bottom: 30px;
+  font-size: 15px;
+}
+
+.button-choose:hover {
```
Comment (mi-chal) [inline src/screens/BeerOfTheDay/styles.scss:24]: Use Sass power

File: src/screens/BeersFavourite/FavouriteBeers.jsx [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+    }
+  }
+  
+  componentDidMount = () => {
+    const beers = getItemFromLocalStorage('beer') || [];
+    this.setState({ beers });
+  } 
+
+  deleteBeer = (id) => {
+
```
Comment (mi-chal) [inline src/screens/BeersFavourite/FavouriteBeers.jsx]: Unnecessary line break

File: src/screens/BeersFavourite/FavouriteBeers.jsx [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+  
+  componentDidMount = () => {
+    const beers = getItemFromLocalStorage('beer') || [];
+    this.setState({ beers });
+  } 
+
+  deleteBeer = (id) => {
+    
+    const beer = this.state.beers.filter(element => element.id !== id);
+
```
Comment (mi-chal) [inline src/screens/BeersFavourite/FavouriteBeers.jsx]: Unnecessary line break

File: src/screens/BeersFavourite/FavouriteBeers.jsx [hunk 11]
```diff
@@ -0,0 +1,55 @@
...
+  }
+  
+  componentDidMount = () => {
+    const beers = getItemFromLocalStorage('beer') || [];
+    this.setState({ beers });
+  } 
+
+  deleteBeer = (id) => {
+    
+    const beer = this.state.beers.filter(element => element.id !== id);
```
Comment (mi-chal) [inline src/screens/BeersFavourite/FavouriteBeers.jsx]: Should be plural

File: src/screens/BeersFavourite/FavouriteBeers.jsx [hunk 12]
```diff
@@ -0,0 +1,55 @@
...
+    
+    this.setState({ beers: beer });
+    saveItemInLocalStorage('beer', beer);
```
Comment (mi-chal) [inline src/screens/BeersFavourite/FavouriteBeers.jsx]: This is not single beer

File: src/screens/BeersFavourite/FavouriteBeers.jsx [hunk 13]
```diff
@@ -0,0 +1,55 @@
...
+      <>
+        <Heading 
+          type="h3"
+          className="title-favourite-beers"
+          text="Your favourite beers"
+        />
+        {this.state.beers.map(({name, id, description, abv, img}) => (
+            <React.Fragment key={id}>
+              <DeleteButton
+              onClick={() => this.deleteBeer(id)} />
```
Comment (mi-chal) [inline src/screens/BeersFavourite/FavouriteBeers.jsx]: No need for moving this to new line

File: src/screens/SeartchBeers/SeartchBeers.jsx [hunk 14]
```diff
@@ -0,0 +1,63 @@
+import React, { PureComponent } from 'react';
+import { fetchAllBeers } from './../../services/beers';
+import Button from '../../shared/components/Button/Button';
+import BeersList from './../../shared/components/BeersList/BeersList';
+import './styles.scss';
+import Heading from '../../shared/components/Heading/Heading';
+
+class SeartchBeersComponent extends PureComponent {
```
Comment (mi-chal) [inline src/screens/SeartchBeers/SeartchBeers.jsx]: Typo

File: src/screens/SeartchBeers/SeartchBeers.jsx [hunk 15]
```diff
@@ -0,0 +1,63 @@
...
+  constructor(props) {
+    super(props);
+    this.state = {
+      beers: null,
+      smallBeer: null
```
Comment (mi-chal) [inline src/screens/SeartchBeers/SeartchBeers.jsx]: What is smallBeer?

File: src/screens/SeartchBeers/SeartchBeers.jsx [hunk 16]
```diff
@@ -0,0 +1,63 @@
...
+
+  weakBeers = () => {
+    const smallvolumeAlcohol = this.state.beers.filter(beer=> beer.abv <= 4.5);
+    this.setState({
+      smallBeer: smallvolumeAlcohol,
+    })
+  }
+  
+  render(){
+    const beers = this.state.beers;
```
Comment (mi-chal) [inline src/screens/SeartchBeers/SeartchBeers.jsx]: Use destructuring

File: src/screens/SeartchBeers/SeartchBeers.jsx [hunk 17]
```diff
@@ -0,0 +1,63 @@
...
+    return (
+      <div className="seartch-page">
+        <Heading 
+          type="h3"
+          className="title-seartch"
+          text='Find Beers'
+        />
+        <h4>Filter: </h4>
```
Comment (mi-chal) [inline src/screens/SeartchBeers/SeartchBeers.jsx]: Use Heading component

File: src/screens/SeartchBeers/SeartchBeers.jsx [hunk 18]
```diff
@@ -0,0 +1,63 @@
...
+            <option>--IBU--</option>
+            <option>less 60</option>
+            <option>more 60</option>
+          </select>
+        </div>
+        <Button 
+          className="button-seartch" 
+          onClick={this.getAllBeers}
+          text="Find beer" />
+        {beers ? <BeersList beers={beers} /> : beers}
```
Comment (mi-chal) [inline src/screens/SeartchBeers/SeartchBeers.jsx]: Again - what will be rendered if there are no beers?

File: src/screens/SeartchBeers/styles.scss [hunk 19]
```diff
@@ -0,0 +1,54 @@
+.seartch-page {
+  display: flex;
+  flex-direction: column;
+  // margin-left: 50px;
+  // margin-right: 50px;
```
Comment (mi-chal) [inline src/screens/SeartchBeers/styles.scss]: Remove

File: src/screens/SeartchBeers/styles.scss [hunk 20]
```diff
@@ -0,0 +1,54 @@
...
+}
+
+.option {
+  margin-bottom: 20px;
+  border-radius: 8px;
+  background-color: white;
+  outline: 0;
+}
+
+.button-seartch {
```
Comment (mi-chal) [inline src/screens/SeartchBeers/styles.scss]: Typo

File: src/screens/SeartchBeers/styles.scss [hunk 21]
```diff
@@ -0,0 +1,54 @@
...
+  width: 75px;
+  height: 75px;
+  border-radius: 50%;
+  outline: 0;
+  font-family: fantasy;
+  border: none;
+  background-color: lightgray;
+}
+
+.button-seartch:hover {
```
Comment (mi-chal) [inline src/screens/SeartchBeers/styles.scss]: Typo. Use Sass

File: src/services/beers.js [hunk 22]
```diff
@@ -0,0 +1,22 @@
...
+export async function fetchRandomBeer() {
+  const url = `${baseUrl}beers/random`
+  try {
+    const response = await fetch(url)
+    return (await response.json())[0]
+  } catch (error) {
+    console.log(error)
+  }
+}
+// $ curl https://api.punkapi.com/v2/beers?page=2&per_page=80
```
Comment (mi-chal) [inline src/services/beers.js]: ?

File: src/shared/components/BeersList/BeersList.jsx [hunk 23]
```diff
@@ -0,0 +1,24 @@
...
+  console.log(beers)
+  return ( 
+    <ul>
+      {beers.map(beer => (
+        <div className="search-beer" key={beer.id}>
+          <h2>{`Name beer: ${beer.name}`}</h2>
+          <h4 className="all-list-beers">{`Description: ${beer.description}`}</h4>
+          <h4 className="all-list-beers">{`Food pairing: ${beer.food_pairing}`}</h4>
+          <h4 className="all-list-beers">{`alcohol vol: ${beer.abv}%`}</h4>
+          <h4 className="all-list-beers">{`IBU: ${beer.ibu}`}</h4>
```
Comment (mi-chal) [inline src/shared/components/BeersList/BeersList.jsx]: Use Heading component

File: src/shared/components/BeersList/BeersList.jsx [hunk 24]
```diff
@@ -0,0 +1,24 @@
+import './styles.scss';
+
+const BeersList = props => {
+  const beers = props.beers;
+  console.log(beers)
```
Comment (mi-chal) [inline src/shared/components/BeersList/BeersList.jsx]: remove

File: src/shared/components/BeersList/BeersList.jsx [hunk 25]
```diff
@@ -0,0 +1,24 @@
+import './styles.scss';
+
+const BeersList = props => {
```
Comment (mi-chal) [inline src/shared/components/BeersList/BeersList.jsx]: Use destructuring

File: src/shared/components/DeleteButton/DeleteButton.jsx [hunk 26]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import './styles.scss';
+
+const DeleteButton = (props) => {
```
Comment (mi-chal) [inline src/shared/components/DeleteButton/DeleteButton.jsx]: Destructuring

File: src/shared/components/FavouriteButton/FavouriteButton.jsx [hunk 27]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faHeart} from '@fortawesome/free-solid-svg-icons';
+import './styles.scss';
+
+const FavouriteButton = (props) => {
```
Comment (mi-chal) [inline src/shared/components/FavouriteButton/FavouriteButton.jsx]: Destructuring

File: src/shared/components/Heading/Heading.jsx [hunk 28]
```diff
@@ -0,0 +1,31 @@
...
+    case H3:
+      return <h3 className={classname}>{text}</h3>;
+    case H2:
+      return <h2 className={classname}>{text}</h2>;
+    default:
+      return <h1 className={classname}>{text}</h1>;
+  }
+};
+
+function Heading (props) {
```
Comment (mi-chal) [inline src/shared/components/Heading/Heading.jsx]: Destructuring

File: src/shared/components/RandomBeer/RandomBeer.jsx [hunk 29]
```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { saveItemInLocalStorage, getItemFromLocalStorage } from '../../../services/localStorage'; 
+import Beer from '../Beer/Beer';
+import FavouriteButton from '../FavouriteButton/FavouriteButton';
+import './styles.scss';
+
+const RandomBeer = props => {
```
Comment (mi-chal) [inline src/shared/components/RandomBeer/RandomBeer.jsx]: Destructuring

File: src/shared/components/RandomBeer/RandomBeer.jsx [hunk 30]
```diff
@@ -0,0 +1,52 @@
...
+//     <div className="row">
+//       <FontAwesomeIcon 
+//         icon={faHeart}
+//         className="icon"
+//         onClick={addBeerToFavourites}
+//       />
+//       <p>Like It!</p>
+//     </div>
+//   </div>
+// }
```
Comment (mi-chal) [inline src/shared/components/RandomBeer/RandomBeer.jsx]: Remove

**Review by mi-chal — CHANGES_REQUESTED** [review]

File: .eslintrc.jsx (hunk 1 above)
Comment (dwantula) [reply .eslintrc.jsx]: Fixed

File: src/screens/BeerOfTheDay/.prettierrc (hunk 5 above)
Comment (dwantula) [reply src/screens/BeerOfTheDay/.prettierrc]: We must took about that.

File: src/screens/BeerOfTheDay/BeerOfTheDay.jsx (hunk 7 above)
Comment (dwantula) [reply src/screens/BeerOfTheDay/BeerOfTheDay.jsx]: When value is empty then display is null

File: src/screens/BeerOfTheDay/styles.scss (hunk 8 above)
Comment (dwantula) [reply src/screens/BeerOfTheDay/styles.scss:24]: Ok

File: src/screens/BeersFavourite/FavouriteBeers.jsx (hunk 9 above)
Comment (dwantula) [reply src/screens/BeersFavourite/FavouriteBeers.jsx]: Done!

File: src/screens/BeersFavourite/FavouriteBeers.jsx (hunk 11 above)
Comment (dwantula) [reply src/screens/BeersFavourite/FavouriteBeers.jsx]: Done!

File: src/screens/BeersFavourite/FavouriteBeers.jsx (hunk 10 above)
Comment (dwantula) [reply src/screens/BeersFavourite/FavouriteBeers.jsx]: Done!

File: src/screens/BeersFavourite/FavouriteBeers.jsx (hunk 12 above)
Comment (dwantula) [reply src/screens/BeersFavourite/FavouriteBeers.jsx]: Fiexed!

File: src/screens/BeersFavourite/FavouriteBeers.jsx (hunk 13 above)
Comment (dwantula) [reply src/screens/BeersFavourite/FavouriteBeers.jsx]: Fixed!

File: src/screens/SeartchBeers/SeartchBeers.jsx (hunk 14 above)
Comment (dwantula) [reply src/screens/SeartchBeers/SeartchBeers.jsx]: Fixed!

File: src/screens/SeartchBeers/SeartchBeers.jsx (hunk 15 above)
Comment (dwantula) [reply src/screens/SeartchBeers/SeartchBeers.jsx]: Fixed!

File: src/screens/SeartchBeers/SeartchBeers.jsx (hunk 16 above)
Comment (dwantula) [reply src/screens/SeartchBeers/SeartchBeers.jsx]: Done!

File: src/screens/SeartchBeers/SeartchBeers.jsx (hunk 17 above)
Comment (dwantula) [reply src/screens/SeartchBeers/SeartchBeers.jsx]: Done!

File: src/screens/SeartchBeers/SeartchBeers.jsx (hunk 18 above)
Comment (dwantula) [reply src/screens/SeartchBeers/SeartchBeers.jsx]: Null

File: src/screens/SeartchBeers/styles.scss (hunk 19 above)
Comment (dwantula) [reply src/screens/SeartchBeers/styles.scss]: Done!

File: src/screens/SeartchBeers/styles.scss (hunk 20 above)
Comment (dwantula) [reply src/screens/SeartchBeers/styles.scss]: Done

File: src/screens/SeartchBeers/styles.scss (hunk 21 above)
Comment (dwantula) [reply src/screens/SeartchBeers/styles.scss]: Done!

File: src/services/beers.js (hunk 22 above)
Comment (dwantula) [reply src/services/beers.js]: Remove

File: src/shared/components/BeersList/BeersList.jsx (hunk 25 above)
Comment (dwantula) [reply src/shared/components/BeersList/BeersList.jsx]: Done!

File: src/shared/components/BeersList/BeersList.jsx (hunk 24 above)
Comment (dwantula) [reply src/shared/components/BeersList/BeersList.jsx]: Done!

File: src/shared/components/DeleteButton/DeleteButton.jsx (hunk 26 above)
Comment (dwantula) [reply src/shared/components/DeleteButton/DeleteButton.jsx]: Done!

File: src/shared/components/FavouriteButton/FavouriteButton.jsx (hunk 27 above)
Comment (dwantula) [reply src/shared/components/FavouriteButton/FavouriteButton.jsx]: Done!

File: src/shared/components/Heading/Heading.jsx (hunk 28 above)
Comment (dwantula) [reply src/shared/components/Heading/Heading.jsx]: Done

File: src/shared/components/RandomBeer/RandomBeer.jsx (hunk 29 above)
Comment (dwantula) [reply src/shared/components/RandomBeer/RandomBeer.jsx]: Done!

File: src/shared/components/RandomBeer/RandomBeer.jsx (hunk 30 above)
Comment (dwantula) [reply src/shared/components/RandomBeer/RandomBeer.jsx]: Done!

File: debug.log (hunk 2 above)
Comment (dwantula) [reply debug.log]: Fixed!

File: src/App.js (hunk 3 above)
Comment (dwantula) [reply src/App.js]: Done!

File: src/App.js (hunk 4 above)
Comment (dwantula) [reply src/App.js]: Done!

File: src/screens/BeerOfTheDay/BeerOfTheDay.jsx (hunk 6 above)
Comment (dwantula) [reply src/screens/BeerOfTheDay/BeerOfTheDay.jsx]: Done!

File: src/shared/components/BeersList/BeersList.jsx (hunk 23 above)
Comment (dwantula) [reply src/shared/components/BeersList/BeersList.jsx]: Done!

File: debug.log [hunk 31]
```diff
@@ -0,0 +1,2 @@
...
+[1118/090755.154:ERROR:directory_reader_win.cc(43)] FindFirstFile: System nie mo�e odnale�� okre�lonej �cie�ki. (0x3)
```
Comment (mi-chal) [inline debug.log]: Remove

File: src/App.jsx [hunk 32]
```diff
@@ -0,0 +1,53 @@
+import React, { PureComponent } from 'react';
+import WorldBeersComponent from './screens/WorldBeers/WorldBeers';
+import BeerOfTheDayComponent from './screens/BeerOfTheDay/BeerOfTheDay';
+import FavouriteBeersComponent from './screens/BeersFavourite/FavouriteBeers';
+import SearchBeersComponent from './screens/SearchBeers/SearchBeers';
+import Navigation from './shared/components/Navigation/Navigation';
+import './App.scss';
+
+const pagesOrScreens = [
```
Comment (mi-chal) [inline src/App.jsx]: So pages or screens? Names must be clear.

File: src/screens/BeerOfTheDay/BeerOfTheDay.jsx [hunk 33]
```diff
@@ -0,0 +1,43 @@
...
+    const { beer } = this.state;
+    return (
+      <div className="beer-day-page">
+        <Heading 
+          type="h3"
+          className="title-beer-day"
+          text="Beer of the day"
+        />
+        <Button 
+          className=" button-choose"
```
Comment (mi-chal) [inline src/screens/BeerOfTheDay/BeerOfTheDay.jsx]: Unnecessary space

File: src/screens/BeerOfTheDay/BeerOfTheDay.jsx [hunk 34]
```diff
@@ -0,0 +1,43 @@
...
+          type="h3"
+          className="title-beer-day"
+          text="Beer of the day"
+        />
+        <Button 
+          className=" button-choose" 
+          onClick={this.getRandomBeer}
+          text="Choose a beer"
+        />
+        {beer ? <RandomBeer value={beer}/> : beer }
```
Comment (mi-chal) [inline src/screens/BeerOfTheDay/BeerOfTheDay.jsx]: No need for ternary here.

File: src/screens/BeerOfTheDay/styles.scss [hunk 35]
```diff
@@ -0,0 +1,33 @@
...
+  background-color: #febc58;
+  box-shadow: -8px -8px 12px 0 rgba(0, 0, 0, 0.3),
+    12px 12px 16px rgba(255, 255, 255, 0.25);
```
Comment (mi-chal) [inline src/screens/BeerOfTheDay/styles.scss]: Oh common. It looks awful - both in the code and on the page.

File: src/screens/BeersFavourite/FavouriteBeers.jsx [hunk 36]
```diff
@@ -0,0 +1,53 @@
...
+class FavouriteBeersComponent extends PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      beers: [],
+    }
+  }
+  
+  componentDidMount = () => {
+    const beers = getItemFromLocalStorage('beer') || [];
```
Comment (mi-chal) [inline src/screens/BeersFavourite/FavouriteBeers.jsx]: Naming indicates that you retrieve single beer from local storage and assign it to array of beers?

File: src/screens/BeersFavourite/FavouriteBeers.jsx [hunk 37]
```diff
@@ -0,0 +1,53 @@
...
+    this.setState({ beers });
+  } 
+
+  deleteBeer = (id) => {
+    const beers = this.state.beers.filter(element => element.id !== id);
+    this.setState({ beers });
+    saveItemInLocalStorage('beer', beers);
```
Comment (mi-chal) [inline src/screens/BeersFavourite/FavouriteBeers.jsx]: Same here

File: src/screens/BeerOfTheDay/BeerOfTheDay.jsx (hunk 34 above)
Comment (mi-chal) [reply src/screens/BeerOfTheDay/BeerOfTheDay.jsx]: Besides that - it should not be separate component - it should be Beer component

File: src/screens/SearchBeers/SearchBeers.jsx [hunk 38]
```diff
@@ -0,0 +1,65 @@
...
+
+  getAllBeers = async () => {
+    const allBeers = await fetchAllBeers()
+    this.setState({
+      beers: allBeers,
+    })
+  }
+
+  weakBeers = () => {
+    const smallvolumeAlcohol = this.state.beers.filter(beer=> beer.abv <= 4.5);
```
Comment (mi-chal) [inline src/screens/SearchBeers/SearchBeers.jsx]: volume cannot be small - it can be low

File: src/screens/SearchBeers/SearchBeers.jsx [hunk 39]
```diff
@@ -0,0 +1,65 @@
...
+    this.setState({
+      smallBeer: smallvolumeAlcohol,
+    })
+  }
+  
+  render(){
+    const beers = this.state.beers;
```
Comment (mi-chal) [inline src/screens/SearchBeers/SearchBeers.jsx]: Use destructuring

File: src/screens/SearchBeers/SearchBeers.jsx [hunk 40]
```diff
@@ -0,0 +1,65 @@
...
+          className="title-search"
+          text='Find Beers'
+        />
+        <Heading 
+          text="Filter"
+          type="h4" />
+        <div className="filter-select">
+          <select className="option">
+            <option>--Alcohol volume--</option>
+            <option onClick={this.weakBeers}>less 4,5 %</option>
```
Comment (mi-chal) [inline src/screens/SearchBeers/SearchBeers.jsx]: So here it is weak beer, there it was small...

File: src/screens/SearchBeers/SearchBeers.jsx [hunk 41]
```diff
@@ -0,0 +1,65 @@
...
+            <option>--IBU--</option>
+            <option>less 60</option>
+            <option>more 60</option>
+          </select>
+        </div>
+        <Button 
+          className="button-search" 
+          onClick={this.getAllBeers}
+          text="Find beer" />
+        {beers ? <BeersList beers={beers} /> : beers}
```
Comment (mi-chal) [inline src/screens/SearchBeers/SearchBeers.jsx]: No need to use ternary here ans separate component. It should be list of Beer component

File: src/screens/SearchBeers/styles.scss [hunk 42]
```diff
@@ -0,0 +1,52 @@
...
+  width: 75px;
+  height: 75px;
+  border-radius: 50%;
+  outline: 0;
+  font-family: fantasy;
+  border: none;
+  background-color: lightgray;
+}
+
+.button-search:hover {
```
Comment (mi-chal) [inline src/screens/SearchBeers/styles.scss:29]: Use Sass

File: src/screens/WorldBeers/WorldBeers.jsx [hunk 43]
```diff
@@ -0,0 +1,15 @@
+import React, { PureComponent } from 'react';
+import './styles.scss';
+
+class WorldBeersComponent extends PureComponent {
+  render() {
+    return (
+      <div>
+        <h3 className="title">Worl Beers</h3>
```
Comment (mi-chal) [inline src/screens/WorldBeers/WorldBeers.jsx]: Use Heading component

File: src/shared/components/Beer/styles.scss [hunk 44]
```diff
@@ -0,0 +1,21 @@
...
+.text-beer {
+  width: 60%;
+  float: right;
+  color: black;
+  text-align: justify;
+  line-height: 35px;
+}
+
+.beer {
+  // border-bottom: 2px solid gray;
```
Comment (mi-chal) [inline src/shared/components/Beer/styles.scss]: Remove

File: src/shared/components/BeersList/BeersList.jsx (hunk 25 above)
Comment (mi-chal) [inline src/shared/components/BeersList/BeersList.jsx]: Use destructuring

File: src/shared/components/BeersList/BeersList.jsx [hunk 45]
```diff
@@ -0,0 +1,24 @@
...
+  const beers = props.beers;
```
Comment (mi-chal) [inline src/shared/components/BeersList/BeersList.jsx]: Oh, common...

File: src/shared/components/BeersList/BeersList.jsx [hunk 46]
```diff
@@ -0,0 +1,24 @@
...
+  
+  return ( 
+    <ul>
+      {beers.map(beer => (
```
Comment (mi-chal) [inline src/shared/components/BeersList/BeersList.jsx]: DE-STRUC-TU-RING

File: src/shared/components/BeersList/BeersList.jsx [hunk 47]
```diff
@@ -0,0 +1,24 @@
...
+        <div className="search-beer" key={beer.id}>
+          <h2>{`Name beer: ${beer.name}`}</h2>
+          <h4 className="all-list-beers">{`Description: ${beer.description}`}</h4>
+          <h4 className="all-list-beers">{`Food pairing: ${beer.food_pairing}`}</h4>
+          <h4 className="all-list-beers">{`alcohol vol: ${beer.abv}%`}</h4>
+          <h4 className="all-list-beers">{`IBU: ${beer.ibu}`}</h4>
+          <div className="row">
+          <img className="img" src={beer.image_url} alt="beer"></img>
```
Comment (mi-chal) [inline src/shared/components/BeersList/BeersList.jsx]: You should not use non pascalCase naming convention in JS

File: src/shared/components/Button/Button.jsx [hunk 48]
```diff
@@ -0,0 +1,14 @@
+import './styles.scss';
+
+const Button = props => {
```
Comment (mi-chal) [inline src/shared/components/Button/Button.jsx]: DE-STRUC-TU-RING

File: src/shared/components/FavouriteButton/FavouriteButton.jsx (hunk 27 above)
Comment (mi-chal) [inline src/shared/components/FavouriteButton/FavouriteButton.jsx]: DE-STRUC-TU-RING

File: src/shared/components/RandomBeer/RandomBeer.jsx [hunk 49]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { saveItemInLocalStorage, getItemFromLocalStorage } from '../../../services/localStorage'; 
+import Beer from '../Beer/Beer';
+import FavouriteButton from '../FavouriteButton/FavouriteButton';
+import './styles.scss';
+
+const RandomBeer = props => {
```
Comment (mi-chal) [inline src/shared/components/RandomBeer/RandomBeer.jsx]: DE-STRUC-TU-RING

File: src/screens/BeerOfTheDay/BeerOfTheDay.jsx [hunk 50]
```diff
@@ -0,0 +1,43 @@
...
+      })
+    }
+  
+
+  render() {
+    const { beer } = this.state;
+    return (
+      <div className="beer-day-page">
+        <Heading 
+          type="h3"
```
Comment (mi-chal) [inline src/screens/BeerOfTheDay/BeerOfTheDay.jsx]: You're exporting heading types from the component so use them everywhere.

**Review by mi-chal — CHANGES_REQUESTED** [review]

File: debug.log (hunk 31 above)
Comment (dwantula) [reply debug.log]: Done!

File: src/App.jsx (hunk 32 above)
Comment (dwantula) [reply src/App.jsx]: Done!

File: src/screens/BeerOfTheDay/BeerOfTheDay.jsx (hunk 33 above)
Comment (dwantula) [reply src/screens/BeerOfTheDay/BeerOfTheDay.jsx]: Done

File: src/screens/BeerOfTheDay/styles.scss (hunk 35 above)
Comment (dwantula) [reply src/screens/BeerOfTheDay/styles.scss]: Done!

File: src/shared/components/Button/Button.jsx (hunk 48 above)
Comment (dwantula) [reply src/shared/components/Button/Button.jsx]: Done

File: src/shared/components/FavouriteButton/FavouriteButton.jsx (hunk 27 above)
Comment (dwantula) [reply src/shared/components/FavouriteButton/FavouriteButton.jsx]: Done!

File: src/shared/components/RandomBeer/RandomBeer.jsx (hunk 49 above)
Comment (dwantula) [reply src/shared/components/RandomBeer/RandomBeer.jsx]: Done!

File: src/screens/WorldBeers/WorldBeers.jsx (hunk 43 above)
Comment (dwantula) [reply src/screens/WorldBeers/WorldBeers.jsx]: Done!

File: src/screens/SearchBeers/SearchBeers.jsx (hunk 38 above)
Comment (dwantula) [reply src/screens/SearchBeers/SearchBeers.jsx]: done!

File: src/screens/SearchBeers/SearchBeers.jsx (hunk 39 above)
Comment (dwantula) [reply src/screens/SearchBeers/SearchBeers.jsx]: Done!

File: src/screens/SearchBeers/SearchBeers.jsx (hunk 40 above)
Comment (dwantula) [reply src/screens/SearchBeers/SearchBeers.jsx]: Done

File: src/shared/components/Beer/styles.scss (hunk 44 above)
Comment (dwantula) [reply src/shared/components/Beer/styles.scss]: Done

File: src/shared/components/BeersList/BeersList.jsx (hunk 25 above)
Comment (dwantula) [reply src/shared/components/BeersList/BeersList.jsx]: Done

File: src/shared/components/BeersList/BeersList.jsx (hunk 45 above)
Comment (dwantula) [reply src/shared/components/BeersList/BeersList.jsx]: Done

File: src/shared/components/BeersList/BeersList.jsx (hunk 46 above)
Comment (dwantula) [reply src/shared/components/BeersList/BeersList.jsx]: Done

File: src/shared/components/BeersList/BeersList.jsx (hunk 47 above)
Comment (dwantula) [reply src/shared/components/BeersList/BeersList.jsx]: Done

File: src/screens/BeerOfTheDay/BeerOfTheDay.jsx (hunk 50 above)
Comment (dwantula) [reply src/screens/BeerOfTheDay/BeerOfTheDay.jsx]: Done

Comment (dwantula) [reply src/screens/BeerOfTheDay/BeerOfTheDay.jsx]: Done!

File: src/screens/BeerOfTheDay/BeerOfTheDay.jsx (hunk 7 above)
Comment (mi-chal) [reply src/screens/BeerOfTheDay/BeerOfTheDay.jsx]: If there is value render <RandomBeer /> if there is not return value?

Comment (dwantula) [reply src/screens/BeerOfTheDay/BeerOfTheDay.jsx]: Yes it return value. I fixed it.

File: src/screens/BeersFavourite/FavouriteBeers.jsx (hunk 36 above)
Comment (dwantula) [reply src/screens/BeersFavourite/FavouriteBeers.jsx]: Fixed!

File: src/screens/BeersFavourite/FavouriteBeers.jsx (hunk 37 above)
Comment (dwantula) [reply src/screens/BeersFavourite/FavouriteBeers.jsx]: fixed

File: src/screens/SearchBeers/SearchBeers.jsx (hunk 41 above)
Comment (dwantula) [reply src/screens/SearchBeers/SearchBeers.jsx]: I deleted BeersList Component

File: src/screens/SearchBeers/styles.scss (hunk 42 above)
Comment (dwantula) [reply src/screens/SearchBeers/styles.scss:29]: Ok

**Pull request merged**